### DISCUSSION TITLE
HTML / CSS / Javascript

### DISCUSSION THREAD

**Discussion by danielehrhardt** [heyform/heyform#141] [Ideas]
It would improve the possibility to build more with Heyform if we could do two things. 

1. Add global HTML / CSS / Javascript Input Settings
2. Add a new Step where you can add HTML / CSS / Javascript

**Comment by iMuFeng** [comment]
I don't understand it. Could you please explain the use case and describe the implementation method?

**Reply by danielehrhardt** [reply]
you can add global html or form step based html
global html use case:
 - tracking script
 - ab test script
 - popup
 - cookie constent
 -  ....
step based html:
 - custom interaction like a info popup 

the possibilities are a huge upgrade

Screenshots from a other Funnelbuilder:
![image](https://github.com/user-attachments/assets/abf357c3-ea9d-4fb5-8e1b-4fa516ecb2be)
![image](https://github.com/user-attachments/assets/6af62762-2a0a-44e2-9912-f22489b1e38d)
